### DISCUSSION TITLE
Gas mask for ESP32

### DISCUSSION THREAD

**Discussion by cristianmitran** [jomjol/AI-on-the-edge-device#1129] [Show and tell]
Hi,

I am sharing a gas mask I use, in case anyone has a similar meter to use it for.
https://www.thingiverse.com/thing:5547444

Cheers,
Cristian

![image](https://user-images.githubusercontent.com/36613624/193839569-a609a345-dc9c-4db6-b088-86ec1dbdbe21.png)
![image](https://user-images.githubusercontent.com/36613624/193839742-0d70bd57-efe5-4baa-a762-7cb055cb238b.png)
![image](https://user-images.githubusercontent.com/36613624/193839850-6cacb0b4-40af-40b9-81c5-ee4283c90656.png)

**Comment by ghost** [comment]
Great! Could you add the type of gasmeter you made it for? Helps others to choose the right model on thingiverse

**Reply by cristianmitran** [reply]
Right. I have added the meter type information.

**Reply by steviehs** [reply]
@cristian: Thanks, great work, unfortuntely not my gas meter brand, so I have design my own and will share it here also. 

Two remarks: 
a) it is always helpful to add the scad file in the thingiverse project. (scad could be even added in this repo) :-)
b) I think going with two parts would make it possible to work against reflections: the base holder and a round piece which fits tight in the base holder for being able to turn the esp32cam free in 360 degrees
 
I guess a section in the Wiki: "Known Meters" or so, would be really great finding such additional information?
